packages/ember-runtime/tests/core/is_array_test.js

```diff
@@ -1,10 +1,10 @@
 import Ember from 'ember-metal/core';
-import { isArray } from 'ember-runtime/utils';
 import ArrayProxy from 'ember-runtime/system/array_proxy';
 
 QUnit.module('Ember Type Checking');
 
 var global = this;
+var isArray = Ember.isArray;
```

Comment:
This test should test the import, then we can confirm the global export in the link I posted above.